flang/include/flang/Parser/parse-tree.h

```diff
@@ -3316,14 +3316,15 @@ struct CompilerDirective {
     TUPLE_CLASS_BOILERPLATE(AssumeAligned);
     std::tuple<common::Indirection<Designator>, uint64_t> t;
   };
+  EMPTY_CLASS(VectorAlways);
```

Comment:
vector is in some compilers a family of directives (vector always, vector aligned etc). Would it be possible to extend the representation to capture all variants.
https://www.intel.com/content/www/us/en/docs/fortran-compiler/developer-guide-reference/2023-0/vector-and-novector.html